src/ui/components/SecondaryToolbox/index.tsx

```diff
@@ -83,6 +80,14 @@ function PanelButtons({
           <div className="label">React</div>
         </button>
       )}
+      <button
```

Comment:
We should wrap this in a feature flag, even if it is `features.network` from `ui/utils/prefs`

Reply:
partly because i dont know if we want to land this without a secondary toolbox for headers and responses

Reply:
Yep, I started adding a setting in the same place that we have other experimental features, and that seemed to be a little involved, so I figured I would bring it up tomorrow and see what people think is the best way to do it.

Reply:
yeah - userSetting is a column in the DB which is its own thing... but features.network would be 5 lines...